backend_e-source/src/main/java/com/github/schulzenrico/esource/services/BookmarkService.java

```diff
@@ -0,0 +1,41 @@
+package com.github.schulzenrico.esource.services;
+
+import com.github.schulzenrico.esource.models.Bookmark;
+import com.github.schulzenrico.esource.models.BookmarkDTO;
+import com.github.schulzenrico.esource.repositories.BookmarkRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class BookmarkService {
+    private BookmarkRepository bookmarkRepository;
+
+    public Bookmark addBookmark(BookmarkDTO bookmarkDTO) {
+       return bookmarkRepository.save(Bookmark.builder()
+                .id(null)
+                .bookmarkUrl(bookmarkDTO.bookmarkUrl())
```

Comment:
```suggestion
       return bookmarkRepository.save(Bookmark.builder()
                .bookmarkUrl(bookmarkDTO.bookmarkUrl())
```

id brauch man nicht extra auf `null` setzen, einfach weglassen, wird dann automatisch auf `null` gesetzt

Reply:
Hatte ich mal ohne. dann kommt die Meldung: 'id' has private access in 'com.github.schulzenrico.esource.models.Bookmark.BookmarkBuilder'